cpp/basix/finite-element.cpp

```diff
@@ -28,6 +28,20 @@ using namespace basix;
 namespace
 {
 //-----------------------------------------------------------------------------
+void orthogonalise(xt::xtensor<double, 2>& wcoeffs)
```

Comment:
A doc comment would be helpful here, e.g. saying that rows of `wcoeffs` are orthogonalised in-place.

Also rows are normalised - would be good to say this.